meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -17,6 +17,7 @@ SRCREV = "c216955d16ca275159891cad296217094d972390"
 
 SRC_URI += "file://amb_allow_sessionbus.patch \
             file://ambd.service \
+            file://ambd_fsa.service \
```

Comment:
You put services on session bus to keep them private to each user, not to keep them private from other apps. I checked through the only AMB docs I could find with a quick searching but I didn't find any mention of the use case of having to launch your own AMB demon with different configuration but only that you might need plugins for additional use cases (e.g getting GPS support). Is that what you're doing? If you could point to relevant AMB docs, that would help a lot.

In the meantime, I've asked @gunnarx to also review here since I believe he knows a bit more about AMB.